githubbot/githubbot/handler.go

```diff
@@ -98,6 +94,11 @@ func (h *Handler) handleSubscribe(cmd string, msg chat1.MsgSummary, create bool,
 	if err != nil {
 		return fmt.Errorf("error splitting command: %s", err)
 	}
+
+	if isSubscribeToBranch(toks) {
```

Comment:
this feels a little small and single-use to be its own util function, maybe put the `len(toks)` condition here and comment what the check means

Reply:
okay sure! and when we plan to add more sub commands to `subscribe` we'll probably refactor this to have better token parsing / pattern matching

Reply:
@Daniel-Wang yeah, once we add event channels we'll need to do something more complex